packages/codec/lib/format/errors.ts

```diff
@@ -1000,3 +1001,16 @@ export interface OverlongArrayOrStringStrictModeError {
 export interface InternalFunctionInABIError {
   kind: "InternalFunctionInABIError";
 }
+
+/**
+ * This doesn't really belong here, but we need something for
```

Comment:
Maybe better to say "This isn't so much a Format error", rather than "here", since the docstring might not show up in the context you expect always

Reply:
Oh, good point.